pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -613,6 +618,32 @@ func handleLastRebootReason(ctx *nodeagentContext) {
 	ctx.lastLock.Unlock()
 }
 
+// handleInstallationLog checks if first-boot file from installation exists
+// remove it and send log from installation to the controller
+func handleInstallationLog(_ *nodeagentContext) {
+	if fileExists(firstbootInstallFile) {
+		err := os.Remove(firstbootInstallFile)
+		if err != nil {
+			log.Errorf("cannot remove installation first-boot file: %s", err)
+		}
+		f, err := os.Open(installLog)
+		if err != nil {
+			log.Errorf("cannot open installation log: %s", err)
+			return
+		}
+		scanner := bufio.NewScanner(f)
+		buf := make([]byte, 0, maxReadSize)
+		scanner.Buffer(buf, maxReadSize)
+		for scanner.Scan() {
+			log.Noticeln(scanner.Text())
```

Comment:
Should we add/replace the source=nodeagent which log adds automatically with source=installer? I don't know if we need to do something special to replace i.e., avoid havig two source atttributes.